### PR TITLE
Update docs to walkthrough completions

### PR DESCRIPTION
**Pull request by bohlmannc**
# What does this PR do?

Adding a completion.md for a more in depth explanation of adding fluvio completions to one's .bashrc file

# Motivation

This PR is the "second half" of issue #1850 in the fluvio repo.

# Related issues

A list of issues closed by or relevant to this PR.

- Closes [#215 ] (https://github.com/infinyon/fluvio-website/issues/215). 

# Additional Notes

# Checklist

- [x] Created or updated all relevant documentation
- [ ] Added unit tests to relevant added or changed code
- [ ] Bumped required crate version numbers
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by sehz** [comment]
Can you link to issue?

**Comment by bohlmannc** [comment]
https://github.com/infinyon/fluvio/issues/1850

**Comment by bohlmannc** [comment]
I can't link to 1850 in fluvio as they are different repositories. I'll create a new issue and link to that.

**Comment by bohlmannc** [comment]
Also, I'm pretty sure I can only link issues using keywords in the actual PR message. The options on the right hand side of this page are greyed out/unavailable to me.

File: content/cli/commands/completion.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Completion
```
Comment (ajhunyady) [inline content/cli/commands/completion.md:2]: We should change to `Command Completion`

**Review by sehz — APPROVED** [review]
LGTM

**Pull request merged**